src/util/entities/VoiceState.ts

```diff
@@ -84,7 +84,7 @@ export class VoiceState extends BaseClass {
 	@Column({ nullable: true })
 	self_stream?: boolean;
 
-	@Column()
+	@Column({ nullable: true })
```

Comment:
This doesn't need to be nullable, it should just be set to false by default. Regardless, you didn't write migrations for this change.

Reply:
> This doesn't need to be nullable, it should just be set to false by default. 
What do you mean by "false by default"? 
> Regardless, you didn't write migrations for this change.
Wasn't sure if I would keep the change but I will write mig's once the first question gets answered.